model/installation_request.go

```diff
@@ -81,9 +82,45 @@ func (request *CreateInstallationRequest) Validate() error {
 		return errors.Wrap(err, "invalid env var settings")
 	}
 
+	return checkSpaces(request)
+}
+
+func checkSpaces(request *CreateInstallationRequest) error {
+	if hasWhiteSpace(request.OwnerID) != -1 {
+		return errors.Errorf("cannot have spaces in owner field. Owner=%s", request.OwnerID)
+	}
+	if hasWhiteSpace(request.DNS) != -1 {
+		return errors.Errorf("cannot have spaces in dns field. DNS=%s", request.DNS)
+	}
+	if hasWhiteSpace(request.Database) != -1 {
+		return errors.Errorf("cannot have spaces in database field. Database=%s", request.Database)
+	}
+	if hasWhiteSpace(request.Filestore) != -1 {
+		return errors.Errorf("cannot have spaces in filestore field. Filestore=%s", request.Filestore)
+	}
+	if hasWhiteSpace(request.Affinity) != -1 {
+		return errors.Errorf("cannot have spaces in affinity field. Affinity=%s", request.Affinity)
+	}
```

Comment:
For all three of these, we have already checked that they are an exact match to a valid value with the methods such as `IsSupportedDatabase`. They should never have spaces so we can drop these checks.

Reply:
ok